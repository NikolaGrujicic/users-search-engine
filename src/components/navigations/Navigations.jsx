import React from 'react';
import Header from './Header';
import Sidebar from './Sidebar';
import useStyles from '../../styles/muiStyles';

const Navigations = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Header />
      <Sidebar />
    </div>
  );
};

export default Navigations;
