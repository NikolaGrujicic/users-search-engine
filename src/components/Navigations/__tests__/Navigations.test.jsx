import React from 'react';
import { shallow } from 'enzyme';
import Navigations from '../Navigations';

test('renders Navigations', () => {
  const wrapper = shallow(
    <Navigations />,
  );
  expect(wrapper).toMatchSnapshot();
});
