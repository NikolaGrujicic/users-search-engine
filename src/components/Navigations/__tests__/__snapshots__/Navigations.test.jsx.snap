// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navigations 1`] = `
<div
  className="makeStyles-root-1"
>
  <TopNav />
  <Sidebar />
</div>
`;
